File: Week-1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height > 8 || height < 1);

    int spaces = height - 1;

    for (int i = 1; i <= height; i++)
    {
        for (int k = spaces; k > 0; k--)
        {
            printf(" ");
        }

        for (int j = 0; j < i * 2; j++)
        {
            if (j == ((i * 2) / 2))
            {
                printf("  ");
                printf("#");
            }
            else
            {
                printf("#");
            }
        }
        printf("\n");
        spaces--;
    }
}

File: Week-6/sentimental-mario-more/mario.py
from cs50 import get_int
while(True):
    height = get_int("Height: ")
    if (height > 0 and height <= 8):
        break

n1 = height - 1
n2 = 2
for i in range(height):
    print(" " * n1, end = "")
    for j in range(n2):
        if(j  == (n2 / 2)):
            print("  ", end = "")
            print("#", end = "")
        else:
            print("#", end = "")
    n1 -= 1
    n2 += 2
    print()


